api/serializers/canteen.py

```diff
@@ -206,6 +211,33 @@ def create(self, validated_data):
 
         return canteen
 
+    def get_central_kitchen_diagnostics(self, obj):
+        if not obj.central_producer_siret:
+            return None
+        try:
+            central_kitchen = Canteen.objects.get(siret=obj.central_producer_siret)
+            diagnostics = central_kitchen.diagnostic_set.filter(
+                central_kitchen_diagnostic_mode__in=[
+                    Diagnostic.CentralKitchenDiagnosticMode.ALL,
+                    Diagnostic.CentralKitchenDiagnosticMode.APPRO,
+                ]
+            )
+            return [
+                {"year": diag.year, "central_kitchen_diagnostic_mode": diag.central_kitchen_diagnostic_mode}
+                for diag in diagnostics
+            ]
+        except Exception:
+            return None
+
+    def get_central_kitchen_name(self, obj):
+        if not obj.central_producer_siret:
+            return None
+        try:
+            central_kitchen = Canteen.objects.get(siret=obj.central_producer_siret)
+            return central_kitchen.name
+        except Exception:
+            return None
```

Comment:
:thinking: Although this would make two requests instead of one (one done by the `exists` and a second one done by the `get`). I prefer to use the `exists` method when there is no need of actually retrieving the object, but we can discuss.